app/Http/Controllers/Api/ApplicationsController.php

```diff
@@ -1151,15 +1206,15 @@ private function create_application(Request $request, $type)
             $application->destination_id = $destination->id;
             $application->destination_type = $destination->getMorphClass();
             $application->environment_id = $environment->id;
-            if (isset($useBuildServer)) {
-                $application->settings->is_build_server_enabled = $useBuildServer;
-                $application->settings->save();
-            }
-
+            
             $application->git_repository = 'coollabsio/coolify';
             $application->git_branch = 'main';
             $application->save();
             $application->refresh();
+            if (isset($useBuildServer)) {
```

Comment:
I will remove this one, because for `dockerimage` based deployments, the build server does not make sense (it is already prebuilt).